## Instructions

It is assumed that the mean systolic blood pressure is μ = 120 mm Hg. In the Honolulu Heart Study, a sample of n = 100 people had an average systolic blood pressure of 130.1 mm Hg with a standard deviation of 21.21 mm Hg. Is the group significantly different (with respect to systolic blood pressure!) from the regular population?

* Set up the hypothesis test.
* Write down all the steps followed for setting up the test.
* Calculate the test statistic by hand and also code it in Python. It should be 4.76190. What decision can you make based on this calculated value?

In [7]:
import scipy.stats as scipy
import math

**Critical value**

In [10]:
tc = +-scipy.t.ppf(1-(0.05/2),df=99)    

In [11]:
tc

-1.9842169515086827

In [4]:
mean_population = 120
mean_sample = 130.1
n = 100
std_sample = 21.21

**Test statistic**

In [5]:
t = (mean_sample - mean_population)/(std_sample/np.sqrt(100))

In [6]:
t

4.761904761904759

We have critical value (95%) in range -1.9842169515086827 to 1.9842169515086827 
Our result is 4.761904761904759 (out of the range), so we **reject null hypothesis**